src/context/RouterContext.tsx

```diff
@@ -50,6 +52,7 @@ const RouterProvider = ({ children }: { children: ReactNode }) => {
     [Routes.ChannelAdded]: ChannelAdded,
     [Routes.ConnectChannels]: ConnectChannels,
     [Routes.SubscriptionFlowEnded]: SubscriptionFlowEnded,
+    [Routes.UserPreferences]: UserPreferences,
```

Comment:
If this is the screen where user is doing the setup then maybe more:
```suggestion
    [Routes.SetupPreferences]: SetupPreferences,
```